components/data/data-anonymize/src/main/java/org/eclipse/dirigible/components/data/anonymize/service/DataAnonymizeService.java

```diff
@@ -346,10 +346,8 @@ public void anonymizeNoSQLColumn(String db, String collection, String column, St
                             break;
                         }
                         case PHONE: {
-                            if (value != null) {
```

Comment:
We should check if this `value` variable can be null at the top. The IDEs mark this line as not necessary but still if `value` can be null, there would be some NPE at the top:
```Java
int length = value.length();
```